### PR TITLE
link article & video posts

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce

/2023/10/24/winging-it-03/
/2023/11/09/unintuitive-objects/

### CODE REVIEW

**Comment by SondraE** [comment]
[Add link to Winging It on Unintuitive Objects post](https://trello.com/c/oUN8SyVd/239-add-link-to-winging-it-on-unintuitive-objects-post)

File: content/blog/2023/unintuitive-objects.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ summary: |
 
 {% import 'embed.macros.njk' as embed %}
 
+> Check out our [Winging It](/wingingit/) channel for a conversation between
+> designers and developers on intuitive web design.
+>
+> **Winging It** episode 3: [Intuitive Web Design with OOUX](/2023/10/24/winging-it-03/)
```
[the remark this answers is not in the document]
Comment (stacyk) [reply content/blog/2023/unintuitive-objects.md:91]: We have these callouts: https://www.oddbird.net/2022/08/18/cq-syntax/ 

I see we have a macro for pull quote but I don't know if there's any styles for it.

[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/blog/2023/unintuitive-objects.md]: Adjusted in https://github.com/oddbird/oddleventy/pull/447/commits/babe3bd9fa73ce59cca7a2e0fed2b62babf7d96f

**Review by jgerigmeyer — APPROVED** [review]

**Review by stacyk — APPROVED** [review]
LGTM now as well!

**Pull request merged**